pkg/testcase/suite.go

```diff
@@ -0,0 +1,28 @@
+package testcase
+
+import (
+	"fmt"
+
+	"github.com/rancher/distros-test-framework/factory"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+const (
+	statusRunning = "Running"
+	nslookup      = "kubernetes.default.svc.cluster.local"
+)
+
+// TestConfigVarVal tests that a given TF variable has the expected string value
+func TestConfigVarVal(g GinkgoTInterface, key, expectedVal string) {
```

Comment:
1) lets try stop adding stuff attached to external stuff ( ginkgo and tf in this case)  the framework is growing and this will become a hell , i can guarante + i can list a lot of reasons to avoid
same for your function  "func GetConfigVarValue("

i was about to write some code and some big text here i can do if you prefer, but if thats ok i would prefer to call you today and talk about and go through.  Or even with the whole time if they want